src/resources/views/fields/password.blade.php

```diff
@@ -5,11 +5,12 @@
     <input
     	type="password"
     	name="{{ $field['name'] }}"
+    	autocomplete="new-password"
```

Comment:
Must have made a mistake here @DelightedD0D . As far as I know, the only possible options for ```autocomplete``` are ```on``` and ```off```. Did I miss something or should this be "off"?